src/util.ts

```diff
@@ -17,3 +20,18 @@ export function mapOwn<T>(
   });
   return arr;
 }
+
+export function toSeries(charts: ChartSeriesMetadata[], rowsGroupedByDate: Map<string, Entry[]>): ChartData[] {
+  const output: ChartData[] = [];
+  charts.forEach(chart => {
+    const data: number[] = Array.from(rowsGroupedByDate.values())
+      .map(rows => rows.map(row => parseInt(row[chart.column])).reduce((a, b) => a + b));
```

Comment:
cool I thought I need to use lodash for reduce. Didn't realized it's built in now.